### ISSUE TITLE
HTTP status codes are not displayed

### ISSUE THREAD

**Issue by thinkjson** [fictive-kin/webshell#19]
This is the output I get when using webshell:

webshell> GET http://www.google.com/
http://www.google.com > GET /
http://www.google.com > 

As you can see, the status code and url are not displayed. I'm using node 0.4.2.

**Comment by thinkjson** [comment]
Let's try that again.

```
webshell> GET http://www.google.com/
http://www.google.com > GET /
http://www.google.com > $_.headers
```

**Comment by scoates** [comment]
Fixed in 0.2; it was displayed, it was just overwritten by the prompt (-:
